api/tests/functional/contacts/destroy.spec.ts

```diff
@@ -0,0 +1,42 @@
+import Database from '@ioc:Adonis/Lucid/Database'
+import { test } from '@japa/runner'
+import Contact from 'App/Models/Contact'
+
+test.group('Contacts destroy', (group) => {
+  // Write your test here
+  group.each.setup(async () => {
+    await Database.beginGlobalTransaction()
+    return () => Database.rollbackGlobalTransaction()
+  })
+
+  test('delete a contact', async ({client, assert}) => {
+
+    // create the contact first
+    const contact = await client.post('/contacts').form({
+      firstName: 'Deborah',
+      surname: 'Okeke',
+      email1: 'testing1.deborah.okeke@gotedo.com',
+      phoneNumber1: '08109210257'
+    })
```

Comment:
Don't make unrelated API request within a test. Use a factory to create the contact.